css/style.css

```diff
@@ -49,23 +67,19 @@ body {
 
 .sidenav {
    height: 100%; /* 100% Full-height */
-   width: 0; /* 0 width - change this with JavaScript */
+   width: 200px;
```

Comment:
@gabrielperales Why is it better to keep the width the same and have it come in from the left?